lib/util/DocumentExtractor.js

```diff
@@ -69,17 +71,27 @@ const extractors = ([
       },
       insertInResult: (documents, request) => {
         if (request.input.action === 'mGet') {
-          request.setResult({}, { status: request.status });
-          request.result.hits = documents;
-          request.result.total = request.result.hits.length;
+          const result = {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "result" hides or potentially hides a variable declared in an outer scope at line 84. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)